apps/standalone/src/app/app.css

```diff
@@ -4,3 +4,10 @@ body,
   height: 100%;
 }
 
+/* 
+  Overwrites style set by OCP's console global CSS
+  Classname is not passed to the root "label" element, we must target the PF switch first
+ */
```

Comment:
shouldnt this change be scoped for ocp plugin only ? how can the standalone app.css affect plugin ?

Reply:
I couldn't figure out a way to properly test this. Using an actual cluster and enabling the ACM operator, the operator modifies the image I'm trying to set in the UI deployment.

Theoretically, I think either loading a base CSS file from a "main" file such as ocp-plugin's AppContext, or wrapping all routes in a common component should work.

Any ideas?

Reply:
using `AppContext` sounds good to me